### PR TITLE
LUN 69: Play TTS for alternative solutions

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/routes/write-exercise/components/WriteExercise.tsx [hunk 1]
```diff
@@ -195,47 +192,60 @@ const AnswerSection = ({
...
-            onFocus={() => setIsFocused(true)}
-            onBlur={() => setIsFocused(false)}
-            onSubmitEditing={checkEntry}
+    <>
+      <ImageCarousel images={document.document_image} />
+      <AudioPlayer
+        document={document}
+        disabled={!result || result === 'similar'}
+        submittedAlternative={submittedAlternative}
+      />
```
Comment (klinzo) [inline src/routes/write-exercise/components/WriteExercise.tsx:208]: This is added, the rest stayed nearly the same.

**Comment by klinzo** [comment]
Please test this. @SteffiStuffel where do we have alternatives?

**Review by SteffiStuffel — CHANGES_REQUESTED** [review]
Just tested this and get errors /unexpected behaviour:

When entering the correct (not alternative) solution, I get an no tts-engine-installed error, although it is installed, but also it is not needed.

When entering a correct alternative solution, the normal audio is played.

E.g. Zollstock has alternative solution Meterstab, it is in the Test-Module (Test -> Level2 -> Test)

**Comment by klinzo** [comment]
> Just tested this and get errors /unexpected behaviour:
> 
> When entering the correct (not alternative) solution, I get an no tts-engine-installed error, although it is installed, but also it is not needed.
> 
> When entering a correct alternative solution, the normal audio is played.
> 
> E.g. Zollstock has alternative solution Meterstab, it is in the Test-Module (Test -> Level2 -> Test)

Should be fixed. Thanks!

**Review by SteffiStuffel — APPROVED** [review]
Tested again, works as expected

**Comment by SteffiStuffel** [comment]
Can you add a release-note as well, please?

**Review by f1sh1918 — APPROVED** [review]
Tested on ios native, nice work :)

**Pull request merged**